content/applications/general/email_communication/email_domain.rst

```diff
@@ -1,37 +1,27 @@
-=======================
-Send an email with Odoo
-=======================
+=========================================
+Configuring DNS Records for Email Sending
+=========================================
 
-Use an email domain in Odoo
-===========================
+SPAM Email
+----------
 
-Documents in Odoo (such as a CRM opportunity, a sales order, an invoice, etc.) have a discussion
-thread, called *chatter*.
+It happens sometimes that some emails from Odoo are misclassified by the different email providers
+and end up in spam folders. At the moment, some settings are out of Odoo's control, notably the
+way the different email providers classify our emails according to their own restriction policy
+and/or limitations.
 
-When a database user posts a message in the chatter, this message is sent by email to the followers
-of the document. If a follower replies to the message, the reply updates the chatter, and Odoo
-relays the reply to the followers as a notification.
+In order for servers to accept emails from Odoo on a more regular basis, one of the solutions is
+for customers to create rules within their own mailbox. A filter can be added to the email inbox so
+that when email is received from the catchall (catchall@yourdatabase.odoo.com) it is moved to the
```

Comment:
There is no email coming from the catchall, 
As the standard (no configuration) for 15, 16 and following, email are received from 
' "name of the user connected" <notifications@dbname.odoo.com> '
Which would be translated by:
' "name of the user connected" <{ICP.mail.from.filter}@{mail.catchall.domain}>'
(where ICP stand for ir.config.parameters, which is the System Parameters as you might now it ;) )